mubench.reviewsite/src/CSVHelper.php

```diff
@@ -0,0 +1,118 @@
+<?php
+namespace MuBench\ReviewSite;
+
+
+class CSVHelper
+{
+
+    public function createCSVFromStats($experiment, $stats)
+    {
+        $rows[] = ["sep=,"];
+        if (strcmp($experiment, "ex1") === 0) {
+            $rows[] =
+                ["detector", "project", "synthetics", "misuses", "potential_hits", "open_reviews", "need_clarification",
+                    "yes_agreements", "no_agreements", "total_agreements", "yes_no_agreements", "no_yes_agreements",
+                    "total_disagreements", "kappa_p0", "kappa_pe", "kappa_score", "hits", "recall"];
+        } elseif (strcmp($experiment, "ex2") === 0) {
+            $rows[] = ["detector", "project", "potential_hits", "open_reviews", "need_clarification", "yes_agreements",
+                "no_agreements", "total_agreements", "yes_no_agreements", "no_yes_agreements", "total_disagreements",
+                "kappa_p0", "kappa_pe", "kappa_score", "hits", "precision"];
+        } else {
+            $rows[] = ["detector", "project", "misuses", "potential_hits", "open_reviews", "need_clarification",
+                "yes_agreements", "no_agreements", "total_agreements", "yes_no_agreements", "no_yes_agreements",
+                "total_disagreements", "kappa_p0", "kappa_pe", "kappa_score", "hits", "recall"];
+        }
+        foreach ($stats as $key => $stat) {
+            $columns = [];
+            $columns[] = $stat->getDisplayName();
```

Comment:
I would prefer to use associative arrays here, because I find `$row["detector"] = $stat->getDisplayName();` much easier to interpret than the current line. This makes constructing the CSV string a little more complex, because you have to use the keys to construct the header row. But you save the repetition in constructing the header rows, ensures headers and values are consistent, and makes things easier to read. The most tricky case is probably the variable number of reviewer columns in the run export below...